src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/UniqueAccessCodeServiceImpl.java

```diff
@@ -57,6 +62,7 @@ public UniqueAccessCodeDTO getUniqueAccessCodeData(String uac) throws CTPExcepti
         if (caseMatch.isPresent()) {
           caseMapperFacade.map(caseMatch.get(), data);
           data.setCaseStatus(CaseStatus.OK);
+          sendRespondentAuthenticatedEvent(data);
```

Comment:
Perhaps the method now needs needs to renamed, as it seems very surprising that a 'get' method has the side effect of sending a message.

Reply:
Renamed method as verbally agreed.